.changeset/@graphql-mesh_plugin-prometheus-7407-dependencies.md

```diff
@@ -0,0 +1,5 @@
+---
+"@graphql-mesh/plugin-prometheus": patch
```

Comment:
```suggestion
"@graphql-mesh/plugin-prometheus": minor
```
I think this should be minor since it is breaking release.

Reply:
it's the automated one. we also have a `major` change in this PR

Reply:
My bad 🤦 I missed it